rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -346,45 +357,59 @@ def _bind_ports_and_wait(self, lports, sandboxes, port_bind_args):
             self._wait_up_port(lports, wait_sync, wait_timeout_s)
 
     @atomic.action_timer("ovn.bind_ovs_vm")
-    def _bind_ovs_port(self, lport, ovs_vsctl, internal=True):
-        port_name = lport["name"]
-        LOG.info("bind %s to %s on %s" % (port_name, sb_name, farm))
-
-        ovs_vsctl.add_port('br-int', port_name, internal=internal)
-        ovs_vsctl.db_set('Interface', port_name,
-                        ('external_ids', {"iface-id":port_name,
-                                            "iface-status":"active"}),
-                        ('admin_state', 'up'))
+    def _bind_ovs_port(self, lport_sbs, internal=True):
+        for sb_name, (sandbox, lports) in lport_sbs.items():
+            farm = sandbox['farm']
+            ovs_vsctl = self.farm_clients(farm, "ovs-vsctl")
+            ovs_vsctl.set_sandbox(sb_name, self.install_method,
+                                  sandbox['host_container'])
+            for lport in lports:
+                port_name = lport["name"]
+                LOG.info("bind %s to %s on %s" % (port_name, sb_name, farm))
+
+                ovs_vsctl.add_port('br-int', port_name, internal=internal)
+                ovs_vsctl.db_set('Interface', port_name,
+                                ('external_ids', {"iface-id":port_name,
+                                                  "iface-status":"active"}),
+                                ('admin_state', 'up'))
+                ovs_vsctl.flush()
```

Comment:
Before the change, each sandbox will flush only once when len(lports) > len(sandboxes). I.e. if there are 10 ports being bound to a sandbox, they are done in one ovsdb transaction. Now with this change, it flushes for every single port binding. Is it on purpose, or shall this line be moved out of this inner for loop?

Reply:
@hzhou8 Ack, I moved the `flush()` in the outer loop.